webpack.config.js

```diff
@@ -1,12 +0,0 @@
-const StylelintPlugin = require('stylelint-webpack-plugin');
-
-module.exports = (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
-    if (!dev || isServer) return config;
-
-    config.plugins.push(new StylelintPlugin({
-        files: ['assets/*.css', 'components/**/*.js', 'pages/**/*.js'],
-        failOnError: false,
-    }));
-
-    return config;
-};
```

Comment:
This file is not needed anymore as the content lives inside of `next.config.js` now.